src/SDK/Language/CLI.php

```diff
@@ -245,6 +245,11 @@ public function getFiles(): array
                 'destination'   => 'lib/commands/push.js',
                 'template'      => 'cli/lib/commands/push.js.twig',
             ],
+            [
+                'scope'         => 'default',
+                'destination'   => 'lib/commands/run.js',
+                'template'      => 'cli/lib/commands/run.js.twig',
+            ],
```

Comment:
You need to add the new template files here

Reply:
Thanks.